packages/forms-web-app/src/config.js

```diff
@@ -43,5 +43,6 @@ module.exports = {
   featureFlag: {
     googleTagManager: process.env.FEATURE_FLAG_GOOGLE_TAG_MANAGER === 'true',
     usePrivateBetaV1RoutesOnly: process.env.PRIVATE_BETA_V1_ROUTES_ONLY === 'true',
+    useSessionStore: process.env.FEATURE_SESSION_STORE === 'true',
```

Comment:
Might be best to rename as `useRedisSessionStore` since the implementation is (needfully) redis-specific.

Reply:
done